Public/Sdk/Public/Prelude/Prelude.OfficeBackCompatHacks.dsc

```diff
@@ -35,6 +35,10 @@ namespace Transformer {
     /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
     export declare function writeAllText(destinationFile: Path, content: string, tags?: string[], description?: string): DerivedFile;
     /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
+    export interface FragmentHandle {}
+    /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
+    export declare function readPipGraphFragment(file: SourceFile, dependencyFragments: FragmentHandle[], description: string): FragmentHandle;

```

Comment:
Please do NOT extend the CompatHacks! If you have to update codegen anyway, please update the codegen to include what the comemnt says above.